package.json

```diff
@@ -12,14 +12,14 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "node src/locales/scripts/get-validated-locales && cross-env ENABLE_AUDIO=true nuxt",
+    "dev": "npm run i18n:get-translations && cross-env ENABLE_AUDIO=true nuxt",
     "build": "npm run i18n:get-translations && nuxt build",
     "generate": "nuxt generate",
     "start": "nuxt start",
     "storybook": "nuxt storybook",
     "test": "jest",
     "test:watch": "jest --collectCoverage=false --watch",
-    "test:unit": "jest --collectCoverage=false",
+    "test:unit": "npm run i18n:get-translations && jest --collectCoverage=false",
```

Comment:
Should this be added to all the `test*` scripts?

Reply:
Not sure it is necessary as this was sufficient to pass tests in the GH action and adding it to `dev` makes it more likely to be local. I didn't want to overcrowd this file but keeping uniformity sounds like a good reason to add it to the other commands as well :)